cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1574,6 +1574,15 @@
   </property>
 
   <property>
+    <name>cdap.ugi.cache.expiration.ms</name>
```

Comment:
Even today, if someone changes the cdap system principal's keytab file, then it will not take effect until the relogin needs to happen, right?